egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -34,6 +34,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
I notice you are still using the name 'ivector_interval' in many places; Vijay asked you to change this to 'ivector_period'.  I also think using 'ivector_period' consistently would be a better idea.  It's OK in principle for the ivector_period used here to be different from the one used while dumping the iVectors.